README.md

```diff
@@ -43,10 +43,12 @@ jobs:
   serial: true
   plan:
   - get: resource-web-app
+  - task: build
+    file: resource-web-app/build.yml
   - put: resource-deploy-web-app
     params:
-      manifest: resource-web-app/manifest.yml
-      path: resource-web-app
```

Comment:
I think I'm misunderstanding how the outputs work. How would the build artifact already be in the outputs? With 0.72 I have to explicitly declare outputs that would be available to other parts of the build plan right? 

Or, are you saying that you'd copy both but just have the manifest point to the binary? My problem with that is just that the manifest would then always assume you had some output directory which seems very concourse specific.